rasa/shared/importers/importer.py

```diff
@@ -56,6 +58,23 @@ def get_config(self) -> Dict:
         """
         raise NotImplementedError()
 
+    def get_model_config(
+        self,
+        cli_par: Dict[Text, Any],
+        training_type: TrainingType = TrainingType.BOTH,
+        is_finetuning: bool = False,
+    ) -> GraphModelConfiguration:
+        """Retrieves model config object.
+
+        Used both by E2EImporter and NLUDataImporter
+        """
+        config = self.get_config()
+        recipe = Recipe.recipe_for_name(config.get("recipe"))
+        model_config = recipe.graph_config_for_recipe(
+            config, cli_par, training_type=training_type, is_finetuning=is_finetuning,
+        )
+        return model_config
```

Comment:
This is causing tests failing because shared is not supposed to import anything from outside (ie. recipes).